possel/application.py

```diff
@@ -82,17 +82,19 @@ def get_etc_file(filename):
 
 def get_arg_parser():
```

Comment:
as long as it has defaults for every value if they aren't set in the config file. Also what happens if there is no config file?